### PR TITLE
Add API to choose output_names when exporting using onnx

### PR DESCRIPTION
**Pull request by lapid92**
## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: model_compression_toolkit/exporter/model_exporter/pytorch/pytorch_export_facade.py [hunk 1]
```diff
@@ -67,6 +68,8 @@ def pytorch_export_model(model: torch.nn.Module,
             PytorchExportSerializationFormat.ONNX).
             quantization_format: Format of how quantizers are exported (fakely-quant, int8, MCTQ quantizers).
             onnx_opset_version: ONNX opset version to use for exported ONNX model.
+            output_names (Optional[List[str]]): Optional list of output node names for export compatibility.
+            This argument is relevant only when using FakelyQuantONNXPyTorchExporter.
```
Comment (ofirgo) [inline model_compression_toolkit/exporter/model_exporter/pytorch/pytorch_export_facade.py]: add warning if not aligned

File: tests_pytest/pytorch_tests/e2e_tests/test_exporter.py [hunk 2]
```diff
@@ -304,6 +318,17 @@ def test_mct_ptq_and_exporter_mctq(self, w_qmethod, abits, a_qmethod, tol):
         self._assert_quant_params_match(quantized_model, onnx_model_dict, a_qmethod, w_qmethod)
         self._assert_outputs_match(quantized_model, self.representative_dataset(1), QuantizationFormat.MCTQ, tol=tol)
 
+    @pytest.mark.parametrize('w_qmethod', [mctq.QuantizationMethod.POWER_OF_TWO])
+    @pytest.mark.parametrize('a_qmethod', [mctq.QuantizationMethod.SYMMETRIC])
+    @pytest.mark.parametrize('abits', [8, 16])
+    @pytest.mark.parametrize('output_names', [None, ['x']])
+    def test_mct_ptq_exporter_mctq_output_names(self, w_qmethod, abits, a_qmethod, output_names):
```
Comment (ofirgo) [inline tests_pytest/pytorch_tests/e2e_tests/test_exporter.py]: add test description comment

File: tests_pytest/pytorch_tests/e2e_tests/test_exporter.py [hunk 3]
```diff
@@ -304,6 +318,17 @@ def test_mct_ptq_and_exporter_mctq(self, w_qmethod, abits, a_qmethod, tol):
...
+        # set_seed(13)
```
Comment (ofirgo) [inline tests_pytest/pytorch_tests/e2e_tests/test_exporter.py]: remove comment

File: tests_pytest/pytorch_tests/e2e_tests/test_exporter.py [hunk 4]
```diff
@@ -304,6 +318,17 @@ def test_mct_ptq_and_exporter_mctq(self, w_qmethod, abits, a_qmethod, tol):
         self._assert_quant_params_match(quantized_model, onnx_model_dict, a_qmethod, w_qmethod)
         self._assert_outputs_match(quantized_model, self.representative_dataset(1), QuantizationFormat.MCTQ, tol=tol)
 
+    @pytest.mark.parametrize('w_qmethod', [mctq.QuantizationMethod.POWER_OF_TWO])
```
Comment (ofirgo) [inline tests_pytest/pytorch_tests/e2e_tests/test_exporter.py]: no need for parametrized

File: tests_pytest/pytorch_tests/e2e_tests/test_exporter.py [hunk 5]
```diff
@@ -304,6 +318,17 @@ def test_mct_ptq_and_exporter_mctq(self, w_qmethod, abits, a_qmethod, tol):
...
+    @pytest.mark.parametrize('a_qmethod', [mctq.QuantizationMethod.SYMMETRIC])
+    @pytest.mark.parametrize('abits', [8, 16])
+    @pytest.mark.parametrize('output_names', [None, ['x']])
+    def test_mct_ptq_exporter_mctq_output_names(self, w_qmethod, abits, a_qmethod, output_names):
+        # set_seed(13)
+        quantized_model = self._run_mct(self.get_model(), self.representative_dataset(1), abits, a_qmethod, w_qmethod)
+        onnx_model_dict = self._run_exporter(quantized_model, self.representative_dataset(1), QuantizationFormat.MCTQ,
```
Comment (ofirgo) [inline tests_pytest/pytorch_tests/e2e_tests/test_exporter.py:335]: add similar tests for onnx fakely (no MCTQ)
